src/metadata_crawler/api/config.py

```diff
@@ -178,38 +179,45 @@ class ConfigMerger:
     """
 
     def __init__(
-        self,
-        user_path: Optional[
-            Union[Path, str, Dict[str, Any], tomlkit.TOMLDocument]
-        ] = None,
+        self, *user_paths: Union[Path, str, Dict[str, Any], tomlkit.TOMLDocument]
     ):
         # parse both documents
         system_path = Path(__file__).parent / "drs_config.toml"
         self._system_doc = tomlkit.parse(system_path.read_text(encoding="utf-8"))
-        _config: str = ""
-        if user_path is not None:
+        _configs: List[str] = []
+        for user_path in user_paths:
             if isinstance(user_path, (str, Path)) and os.path.isdir(user_path):
-                _config = (
-                    Path(user_path).expanduser().absolute() / "drs_config.toml"
-                ).read_text(encoding="utf-8")
+                _configs.append(
+                    (
+                        Path(user_path).expanduser().absolute()
+                        / "drs_config.toml"
+                    ).read_text(encoding="utf-8")
+                )
             elif isinstance(user_path, (str, Path)) and os.path.isfile(user_path):
-                _config = (
-                    Path(user_path)
-                    .expanduser()
-                    .absolute()
-                    .read_text(encoding="utf-8")
+                _configs.append(
+                    (
+                        Path(user_path)
+                        .expanduser()
+                        .absolute()
+                        .read_text(encoding="utf-8")
+                    )
                 )
             elif isinstance(user_path, (str, Path)):
-                _config = str(user_path)
+                paths = glob.glob(str(user_path), recursive=False) or [
+                    str(user_path)
+                ]
+                for path in paths:
+                    if os.path.isfile(path):
+                        _configs.append(Path(path).read_text(encoding="utf-8"))
+                    else:
```

Comment:
I don't understand this case. What happens if `user_path` resolves to a directory? If we assume `user_path=/path/to/*` and this contains some subfolders, we would end up with appending a subfolder to `_configs`, no?